pkg/cmd/cluster/bind/bind.go

```diff
@@ -92,7 +97,7 @@ func runBind(opts *Options) error {
 	// In future config will include Id's of other services
 	if cfg.Services.Kafka == nil || opts.ignoreContext {
 		// nolint
-		selectedKafka, err := kafka.InteractiveSelect(connection, logger)
+		selectedKafka, err := kafka.InteractiveSelect(apiConnection, logger)
```

Comment:
It is possible that `selectedKafka` will be nil if there are no Kafka instances available.

Same issue caused a nil-pointer error in the `use` command: https://github.com/redhat-developer/app-services-cli/pull/602/files